planb/PBPackageInstaller.m

```diff
@@ -24,37 +24,37 @@
 #import "PBLogging.h"
 #import "PBPackageInstaller.h"
 
-@implementation PBPackageInstaller
+@interface PBPackageInstaller ()
 
-+ (NSString *)SHA1ForFileAtPath:(NSString *)path {
-  unsigned char sha1[CC_SHA1_DIGEST_LENGTH];
-  NSData *fileData = [NSData dataWithContentsOfFile:path
-                                            options:NSDataReadingMappedIfSafe
-                                              error:nil];
+/// Path to mounted disk image, e.g. '/tmp/planb-pkg.duc3eP'.
+@property(nonatomic, copy) NSString *mountPoint;
 
-  CC_SHA1([fileData bytes], (unsigned int)[fileData length], sha1);
-  NSMutableString *buf = [[NSMutableString alloc] initWithCapacity:CC_SHA1_DIGEST_LENGTH * 2];
+/// Path to temporary dmg file, e.g. '/tmp/planb-dmg.ihI1UV/pkg-stable.dmg'.
+@property(nonatomic, copy) NSString *packagePath;
 
-  for (int i = 0; i < CC_SHA1_DIGEST_LENGTH; i++) {
-    [buf appendFormat:@"%02x", (unsigned char)sha1[i]];
-  }
+/// Package receipt name, e.g. 'com.megacorp.corp.pkg'.
+@property(nonatomic, copy) NSString *receiptName;
```

Comment:
nit: can you declare this property before packagePath. That way it matches the order or the arguments for the init method below.